config/packages/security.yaml

```diff
@@ -46,7 +46,7 @@ security:
 
     access_control:
         - { path: ^/api/programmes, roles: ROLE_ADMIN }
-        - { path: ^/api/users, roles: ROLE_ADMIN }
+        - { path: ^/api/users, roles: PUBLIC_ACCESS }
```

Comment:
this is not a good thing. it's ok if you have public access for register endpoint (so POST), but if you have a GET on users, it's not ok for everybody to access it